app/react/src/client/preview/render.js

```diff
@@ -19,12 +19,19 @@ if (isBrowser) {
   rootEl = document.getElementById('root');
 }
 
+function render(node, el) {
+  ReactDOM.render(
+    process.env.STORYBOOK_EXAMPLE_APP ? <React.StrictMode>{node}</React.StrictMode> : node,
```

Comment:
Maybe we should add this as a setting so users can also enforce strictness in their storybooks? Either `yarn storybook --strict` or `import { setStrict } from '@storybook/react'; setStrict(true)`.

I'd probably lean toward the second as it is react only.

Reply:
It's quite trivial to add on user side:
```
addDecorator(story => <StrictMode>{story()}<StrictMode>)
```

Reply:
Oh, fair enough!